proto/file.proto

```diff
@@ -36,6 +37,11 @@ message CreateUploadRequest {
     int64 size = 5;
 }
 
+message UpdateFilesResponse {
+	repeated string updated = 1;
+	repeated string failed = 2;
```

Comment:
Let's keep a standard of 4 spaces instead of tabs

Reply:
Let's not.

Reply:
This is the standard throughout the service. we're not going to change that if it's not a refactor sprint